src/deepsparse/benchmark_model/README.md

```diff
@@ -14,103 +14,123 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Benchmarking
+# Benchmarking 📜
```

Comment:
```suggestion
# Benchmarking ONNX Models
```

Reply:
It's not clear to me if one is Benchmarking [whatever that icon is] but at minimum, always good to provide some clarity in a doc headline.
Also noting to self/the royal we, if we convert this to Docs at some point, not understood yet how icons convert as we don't have that as part of a style/messaging guide.